src/platforms/react-native/manual-setup/manual-setup.mdx

```diff
@@ -8,31 +8,38 @@ description: "Learn about manual configuration for iOS and Android."
 
 If you can’t (or prefer not to) run the [automatic setup](/platforms/react-native), use the instructions below to configure your application.
 
-<Alert level="info" title="Prerequisite">
+## Install SDK Package
 
-Before you can begin, you need to create a `sentry.properties` file at the root of your React Native _android_ and _ios_ project directories. Check out these examples of property files for [iOS](https://github.com/getsentry/examples/blob/master/react-native/AwesomeProject/ios/sentry.properties) and [Android](https://github.com/getsentry/examples/blob/master/react-native/AwesomeProject/android/sentry.properties).
+Install the `@sentry/react-native` package:
 
-</Alert>
+```bash {tabTitle:Yarn}
+yarn add @sentry/react-native
+```
+
+```bash {tabTitle:npm}
+npm install --save @sentry/react-native
+```
```

Comment:
I believe the convention is to have npm before yarn - can you swap these?

Reply:
I'd give you a suggested edit, but Github doesn't do well with code blocks in suggested edits :/